components/forms/Toggle/style.module.scss

```diff
@@ -0,0 +1,68 @@
+@import "styles/index.scss";
+
+.toggle {
+  $size: 14px;
+
+  align-items: center;
+  align-content: center;
+  position: relative;
+  height: $size;
+  width: $size;
+  flex-shrink: 0;
+
+  &__switch {
+    position: relative;
+    display: inline-block;
+  }
+
+  &__input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+  }
+
+  &__slider {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: $color-gray;
+    -webkit-transition: 0.4s;
+    transition: 0.4s;
+    height: 8px;
+    width: 24px;
+    border-radius: 34px;
+  }
+
+  &__input:checked + &__slider {
+    background: $color-gray;
+  }
+
+  &__input:focus + &__slider {
+    box-shadow: 0 0 1px $color-onyx;
```

Comment:
The slider has an `inset` `box-shadow` using a transparent background color. I don't see `inset` anywhere. I think it may be missed or you used a different technique. I would double check the appearance on this, as @simonwebdesign won't pass it.